pkg/kind/kind.go

```diff
@@ -85,8 +87,8 @@ func createKindCluster() error {
 	time.Sleep(10 * time.Second)
 
 	clusterWait := exec.Command("kubectl", "wait", "pod", "--timeout=-1s", "--for=condition=Ready", "-l", "!job-name", "-n", "kube-system")
-	if err := clusterWait.Run(); err != nil {
-		return fmt.Errorf("wait: %w", err)
+	if err := util.RunCommand(clusterWait); err != nil {
+		return fmt.Errorf("%w", err)
```

Comment:
this is a no-op now, might as well just do `return err`

Reply:
same for [line 79](https://github.com/psschwei/kn-plugin-quickstart/blob/include-command-output/pkg/kind/kind.go#L78-L79)?

Reply:
yeah, there's no point doing fmt.Errorf if you're not changing the error in any way

Reply:
cool -- will need to clean up the error chains at some point (not in this PR though probably), some of them end up looking like "ERROR: place 1: place 2: place 3: actual error code"

Reply:
yeah welcome to Go ;) it's ok to only wrap the error where you're introducing useful context